tools/iana/makefile

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
All other makefiles in the tree are named `Makefile`, I think that makes sense for this too.